numpy/core/fromnumeric.py

```diff
@@ -1111,6 +1120,16 @@ def searchsorted(a, v, side='left', sorter=None):
     As of NumPy 1.4.0 `searchsorted` works with real/complex arrays containing
     `nan` values. The enhanced sort order is documented in `sort`.
 
+    This function is a vectorized version of the C++ ``std::lower_bound`` and
```

Comment:
Not sure this helps. I expect few numpy users are familiar with the C++ standard library. These functions also return iterators, which is a bottomless swamp in the C++ landscape, filled with the corpses of the unwary.

Reply:
Agree with @charris here, don't think this helps in a docstring.

Reply:
I'd like this to be somewhere to that `numpy upper_bound` gives google results. I'm ok with making it much less prominent, but I'd like to keep it - especially since the C++ explain the functionality in a very different way, so it may not be obvious that they are equivalent.

Reply:
On second thoughts, referring to the builtin `bisect` module would be more useful.